src/test/java/com/easypost/InsuranceTest.java

```diff
@@ -58,11 +58,15 @@ public void testCreate() throws EasyPostException {
      */
     @Test
     public void testRetrieve() throws EasyPostException {
-        Insurance retrievedInsurance = Insurance.retrieve(globalInsurance.getId());
+        _vcr.setUpTest("retrieve");
+
+        Insurance insurance = createBasicInsurance();
+
+        Insurance retrievedInsurance = Insurance.retrieve(insurance.getId());
 
-        assertTrue(retrievedInsurance instanceof Insurance);
-        assertTrue(retrievedInsurance.getId().startsWith("ins_"));
-        assertEquals(globalInsurance.getId(), retrievedInsurance.getId());
+        assertInstanceOf(Insurance.class, insurance);
+        // Must compare IDs since other elements of object may be different
```

Comment:
May be able to compare the whole thing?

Reply:
Cannot currently do whole-object comparisons in Java.